statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -213,8 +213,19 @@ func (wr *WidgetsRender) prepareBlockInfo() {
 	consensusState := wr.getFromCacheAsString(core.MetricConsensusState)
 	rows[5] = []string{fmt.Sprintf("Consensus state: %v", consensusState)}
 
-	consensusRoundState := wr.getFromCacheAsString(core.MetricConsensusRoundState)
-	rows[6] = []string{fmt.Sprintf("Consensus round state: %v", consensusRoundState)}
+	syncStatus := wr.getFromCacheAsUint64(core.MetricIsSyncing)
+	switch syncStatus {
+	case 1:
+		rows[6] = []string{fmt.Sprintf("Consensus round state: Not available (syncing)")}
```

Comment:
Might use N/A instead of Not available to spare some space

Reply:
Done.